packages/patterns-react/src/patterns/LeadSpace/LeadSpace.js

```diff
@@ -23,12 +23,17 @@ const { prefix } = settings;
  * @returns {*} button components
  */
 function renderButtons(buttons) {
+  const iconMap = new Map([
+    ['ArrowRight', ArrowRight20],
+    ['ArrowDown', ArrowDown20],
+    ['Pdf', Pdf20],
+  ]);
```

Comment:
hmm yeah was making things complicated - changed to `obj` 

Not sure if we could simplify to `renderIcon={button.renderIcon + '20'}` because I'm importing the icons and they need to be called somewhere else lint will complain

Reply:
hmm yeah was making things complicated - changed to obj

Not sure if we could simplify to renderIcon={button.renderIcon + '20'} because I'm importing the icons and they need to be called somewhere else lint will complain

Reply:
hmm yeah was making things complicated - changed to obj

Not sure if we could simplify to renderIcon={button.renderIcon + '20'} because I'm importing the icons and they need to be called somewhere else lint will complain